app/src/main/java/com/courseproject/tindar/entities/Conversation.java

```diff
@@ -6,20 +6,20 @@
 public abstract class Conversation {
     private String user1;
     private String user2;
-    private ArrayList<MessageInterface> messages;
+    private ArrayList<MessageModel> messages;
```

Comment:
I still don't get the ArrayList of the interface. The interface doesn't hold attributes. So what are we making a list of?

Reply:
The thing that implements the interface holds attributes.